website/src/docs/experiments/env-precedence.md

```diff
@@ -1,31 +1,27 @@
 ---
-slug: '/experiments/env-precedence'
+title: "Env Precedence (#1038)"
+description: Expérimentation pour changer la précédence des variables d'environnement dans Task
+outline: deep
 ---
 
 # Env Precedence (#1038)
 
-:::caution
-
+::: warning
 All experimental features are subject to breaking changes and/or removal _at any
 time_. We strongly recommend that you do not use these features in a production
 environment. They are intended for testing and feedback only.
-
 :::
 
-:::warning
-
+::: danger
 This experiment breaks the following functionality:
 
 - environment variable will take precedence over OS environment variables
+  :::
 
-:::
-
-:::info
-
+::: info
 To enable this experiment, set the environment variable:
 `TASK_X_ENV_PRECEDENCE=1`. Check out [our guide to enabling
-experiments][enabling-experiments] for more information.
-
+experiments](/experiments/#enabling-experiments) for more information.
 :::
```

Comment:
I mentioned this on Discord already and its not that important, but if its possible, it would be nice to re-add the icons we had on these admonitions on the old site. They look a bit plain without them.

Reply:
<img width="723" height="433" alt="image" src="https://github.com/user-attachments/assets/483d808c-6b99-4af0-a264-23dcf5fe5dbb" />


Dark mode : 
<img width="737" height="435" alt="image" src="https://github.com/user-attachments/assets/b0e29fd4-143e-4d35-9b3f-8f2107913b43" />

Disclaimer, I am not a designer (and I have really bad taster in termes of design) so feel free to comment it